frappe/core/api/user_invitation.py

```diff
@@ -0,0 +1,110 @@
+import frappe
+
+USER_INVITATION_DOCTYPE = "User Invitation"
+FRAMEWORK_APP_NAME = " "
+
+
+@frappe.whitelist()
+def invite_by_email(
+	emails: str, role: str, redirect_to_path: str, app_name: str = FRAMEWORK_APP_NAME
```

Comment:
Avoid use of `" "` as sentinal value for default. You can use `None` as sentinal value here?